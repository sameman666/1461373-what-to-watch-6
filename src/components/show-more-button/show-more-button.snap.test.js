import React from 'react';
import {render} from '@testing-library/react';
import ShowMoreButton from './show-more-button';

it(`Should ShowMore button render correctly`, () => {
  const {container} = render(<ShowMoreButton />);
  expect(container).toMatchSnapshot();
});
