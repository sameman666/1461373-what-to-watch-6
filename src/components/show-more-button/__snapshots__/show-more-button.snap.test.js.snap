// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should ShowMore button render correctly 1`] = `
<div>
  <div
    class="catalog__more"
  >
    <button
      class="catalog__button"
      type="button"
    >
      Show more
    </button>
  </div>
</div>
`;
